### DISCUSSION TITLE
Too much space in between elements

### DISCUSSION THREAD

**Discussion by kyungnapark** [thegetty/quire#168] [Get Help] [answered]
My issue is that there is way too much space in between the title and the element on each page. 

Ideally I'd like to limit each page to one image/block of text. 

However, with dimensions of 7x7 the spacing of the title and image is thrown off. (It works okay with the default 8.5x11) 

Is this something I have to manually adjust through variables.scss/print.scss?

I'm also noticing there is a huge chunk of space around the title, and I'm wondering if that's also a css issue.

Comments/input would be much appreciated. :) 

![Screen Shot 2021-08-30 at 5 13 27 PM](https://user-images.githubusercontent.com/87673056/131421681-932aa764-e2ab-4f95-9569-5581ae808d87.png)
![Screen Shot 2021-08-30 at 5 17 08 PM](https://user-images.githubusercontent.com/87673056/131422388-76c17883-50f8-4f09-9e27-57b84d02c875.png)

**Comment by geealbers** [comment, marked as answer]
Hi @pancaketents. Yes this is all CSS and I believe that rather than being able to adjust it through SCSS variables, you will instead have to write some custom CSS to override things and set them up the way you'd like. Unfortunately, it's not a particularly easy process because it mostly means making changes to the CSS, outputting the PDF to check if it worked, making more changes, outputting again, and so on and so on. We're hoping to one day move to a new PDF generator that will alleviate the problem, but for now, we're stuck with what we've got. 

In determining what CSS has to be modified, the best way to do it is to preview it in a browser and their are instructions for that here: https://quire.getty.edu/documentation/multiformat-output/#display-the-pdf-version-in-a-browser. The one big thing you'll notice is that while the browser preview done in this way is closer to the PDF output, it's not exact at all (for instance the page size isn't rendered in the browser) so you'll sill have to do a lot of back and forth with outputting the PDF and tweaking the CSS back and forth.

Hope this helps a little.

**Reply by kyungnapark** [reply]
Gotcha, thanks Greg!
